.github/workflows/build.yml

```diff
@@ -33,6 +34,7 @@ jobs:
           echo "timestamp=$timestamp" >> $GITHUB_OUTPUT
 
   base:
+    if: (github.event_name != 'schedule' || github.repository == 'discourse/discourse_docker')
```

Comment:
I'm not sure I follow here. Why are we skipping building of the base image when it is a scheduled job?

Reply:
skipping when it's a scheduled job and is a fork - this prevents a bunch of jobs on new forks until actions are disabled but will remove for now from the other comment.